tests/jsTests/menuSpec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Suite : Tests for showing and hiding of menu
+ * Description : This is a test suite for showing and hiding menu for smaller screens.
```

Comment:
What is the standard way of writing comments in javascript ? No extra information, useful to another developer is given here, "suite"", "description" and "describe" all say the same thing, in different language. What do you mean by "smaller" screen ? Please be more explicit. Purpose of comments is to make the code more understandable by another developer. dont write redundant repetitive facts.